rust/agama-lib/share/profile.schema.json

```diff
@@ -183,6 +183,40 @@
             "type": "string",
             "examples": ["vim"]
           }
+        },
+        "extraRepositories": {
+          "title": "List of user specified repositories that will be used on top of default ones",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "required": ["alias", "url"],
```

Comment:
The alias could be optional, we could generate some random unique ugly name if missing. But I'm not sure how that would be used by users.

Personally I'd like to specify a well known meaningful name instead of some random one.